### DISCUSSION TITLE
Custom `my` value

### DISCUSSION THREAD

**Discussion by gasatrya** [tailwindlabs/tailwindcss#10380] [Help] [answered]
Hi there,
So, currently, I need to reuse `margin-top: 0` and `margin-bottom: 1.5rem` anywhere in my site. I know I can extend the margin value using `spacing`, but how to extend `my`? I tried:

```js
extend: {
    // spacing:  or
    margin: {
      custom: ['0 1.5rem'], // 1
      custom: ['0', '1.5rem'], // 2
    },
  },
},
```

None of them worked. Any help would be appreciate.

**Comment by gasatrya** [comment]
I know I can just do `@apply mb-6 mt-0` in the CSS file, but I think customizing via `extend` is more Tailwind ways.

**Comment by wongjn** [comment, marked as answer]
The core plugin margin utilities expect a single value, so you would not be able to achieve this with an `extend` value alone. You could look at writing a plugin that adds it as a new utility class.

**Reply by gasatrya** [reply]
I see, thanks for the anwers.
